### ISSUE TITLE
 Error: Failed to add page binding with name sendToHaxroomie

### ISSUE THREAD

**Issue by morko** [morko/haxroomie#7]
Happened when closed and opened the room fast. Browser probably did not have enough time to navigate out of the haxball page.
```
2019-02-21 10:39:52 [error]: SEND_TO_ROOM_ERROR (haxroomie): Error: Failed to add page binding with name sendToHaxroomie: window['sendToHaxroomie'] already exists!                     at Page.exposeFunction (C:\Users\oskar\workspace\node\haxroomie\node_modules\puppeteer\lib\Page.js:428:13)                                                                          at Page.<anonymous> (C:\Users\oskar\workspace\node\haxroomie\node_modules\puppeteer\lib\helper.js:109:23)                                                                           at RoomOpener.open (C:\Users\oskar\workspace\node\haxroomie\src\room\RoomOpener.js:144:21)                                                                                          at process._tickCallback (internal/process/next_tick.js:68:7)                                                              
```

**Comment by morko** [comment]
This happens because sendToHaxroomie is a PAGE binding so it does not get destroyed on navigating out of the domain.

**Comment by morko** [comment]
fixed in d930fdc
